include/range/v3/range_fwd.hpp

```diff
@@ -25,6 +25,29 @@
 #include <range/v3/utility/static_const.hpp>
 #include <range/v3/version.hpp>
 
+// if the standard library ships <ranges>, play nice with it
+#if __has_include(<version>)
```

Comment:
`__has_include` is C++17. range-v3 still supports C++14. Please guard this with `#if defined(__has_include)`.